src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -427,10 +430,10 @@ impl<S: LocalStateStore> WriteFuture<S> {
         stream: BoxedMessageStream,
         write_state: LogStoreWriteState<S>,
     ) -> Self {
-        let instant = Instant::now() + duration;
-        tracing::trace!(?instant, ?duration, "write_future_pause");
+        tracing::trace!(now = ?Instant::now(), ?duration, "write_future_pause");
```

Comment:
Seems redundant to call `Instant::now()` in consecutively 3 lines. Can call once and reuse later.